mitmproxy/addons/serverplayback.py

```diff
@@ -30,7 +30,15 @@ def load(self, loader):
             "server_replay_kill_extra",
             bool,
             False,
-            "Kill extra requests during replay (for which no replayable response was found).",
+            "Kill extra requests during replay (for which no replayable response was found)."
+            "[Deprecated, prefer to use server_replay_extra='kill']",
```

Comment:
Awesome! In `configure`, can we emit a `logger.warning` message if server_replay_kill_extra is set so that people start moving over? :)